### ISSUE TITLE
Broken in WordPress 4.7?

### ISSUE THREAD

**Issue by sistercylon** [Automattic/amp-wp#606]

**Comment by sistercylon** [comment]
I see that the plugin shows "not tested with your version of WordPress" in the plugin directory. I'm running 4.7. I created a custom template and styles, added the necessary hooks and filters in the functions.php file of my child theme, and things were working fine. My pages validated with AMP and my custom styles were showing up (as of October 25, 2106, after the last updated of this plugin). Now we're finally ready to launch our AMP functionality on our live site, and my custom templates don't work. None of my styles are pulling through, my custom related posts aren't showing up, featured image isn't showing up (I had added this in my custom AMP template). This is true on my local installation and my dev site as well as our live site. Anyone else having issues like this?

**Comment by sdk7327** [comment]
I'm having this same problem. Did you happen to find a fix?

**Comment by sistercylon** [comment]
Hi there @sdk7327, I don't know what changed in WordPress during the update but I think it had something to do with permalink settings. It seems like the update caused my custom permalink to reset to default. It broke other things on my site, too. So I had to reapply my permalink settings, and I had to update my custom AMP templates to match. I know that's not very specific but hopefully it helps.
